source/man/option-u-user.inc

```diff
@@ -0,0 +1,6 @@
+**-u**\ *user/mask*
+   | Run the *command* only for the given *user*. It's also possible to
+     use '**\***' and '**?**' wildcards (e.g. -u \*@example.org).
+   | When neither the **-A** option, nor the **-F**\ *file* option, nor
+     the **-u**\ *user* was specified, the *command* will be executed
+     with the environment of the currently logged in user.
```

Comment:
Leading lines with `|` does not yield the same output as with the previous manpages. This creates a line block that persists initial indents and linebreaks. I don't think there is a good way to break this onto two lines without creating two separate paragraphs.

Also remove the `\` that prepend whitespace that should exist in the output (e.g. after `**-F**`).
Also `*@example.org` is unhighlighted in the existing manpages but underlined in this version.